stable/database/files/nuosm

```diff
@@ -1,22 +1,31 @@
-#!/bin/sh
+#!/bin/bash
 
-[ -n "$NUOSM_LINT" ] && set -n
+[ "$NUODB_DEBUG" = "verbose" ] && set -x
+[ "$NUOSM_VALIDATE" = "true" ] && set -e
```

Comment:
If we exit immediately, then we are not clearing some important metadata as `first_req` and `$startup_key/$DB_NAME`. Would that cause some problems?